package.json

```diff
@@ -53,6 +53,7 @@
   "dependencies": {
     "@elastic/elasticsearch": "7.11.0",
     "@elastic/elasticsearch-mock": "0.3.0",
+    "@senecacdot/satellite": "1.8.0",
```

Comment:
I don't want to use Satellite in the monolithic back-end, only in the microservices.  Let's undo this.